workspace/dev-compile/src/rollup/rollup.ts

```diff
@@ -63,14 +63,11 @@ function compilePackage(pkg: PackageInfo, options: CompileOptions): RollupOption
       PLUGINS.push(importMeta(mode));
     }
 
-    const deps = Object.keys(pkg.dependencies);
-
     const entries = entryPoints(pkg, mode).map((options) => ({
       ...options,
```

Comment:
```suggestion
      ...options,
      treeshake: { moduleSideEffects: false },
```

Reply:
This is handled in the `entryPoint` function